static/css/_panel.scss

```diff
@@ -299,6 +299,9 @@ $disabled-color: #F2F2F2;
         font-size: 11px;
         border-bottom: 1px solid #f2f2f2;
         border-right: 1px solid #f2f2f2;
+        padding-left: 8px !important;
```

Comment:
**Suggestion** to use `rem` instead of `px` since `px` is not scalable. Here, `8px` would be `0.8rem`

```suggestion
        padding-left: 0.8rem !important;
```